server/tests/src/test/java/org/infinispan/server/footprint/FootprintIT.java

```diff
@@ -30,8 +30,8 @@ public class FootprintIT {
    private static final int LOADED_CLASS_COUNT_UPPER_BOUND = 11_500;
    private static final long HEAP_USAGE_LOWER_BOUND = 23_000_000L;
    private static final long HEAP_USAGE_UPPER_BOUND = 25_000_000L;
-   private static final long DISK_USAGE_LOWER_BOUND = 69_000_000L;
-   private static final long DISK_USAGE_UPPER_BOUND = 74_000_000L;
+   private static final long DISK_USAGE_LOWER_BOUND = 73_000_000L;
+   private static final long DISK_USAGE_UPPER_BOUND = 75_000_000L;
```

Comment:
Is it okay changing the range from 5 MB to 2MB?

Reply:
Disk footprint changes very rarely, and, unfortunately, it decreases even more rarely :)